scripts/src/test/bash/all-tests

```diff
@@ -11,6 +11,10 @@ export DEVON_SKIP_PROJECT_SETUP=true
 for testpath in "$(dirname $0)"/test-*
 do
   testcase="${testpath/*\//}"
+  if [[ ! "${COMMAND_TEST}" && "${testcase}" == "test-sfcommands" ]]
+  then
+    continue
+  fi
```

Comment:
See previous comment: IMHO we should change this.
I see that we should have many small integration tests and therefore maintaining a list of filenames for them here is not good for maintenance.
We could simply use the triggering variable to tweak the glob-pattern (e.g. `test-*` vs. `*test-*`).

IMHO we should better name the triggering variable `INTEGRATION_TEST` or `NIGHTLY_TEST` rather than `COMMAND_TEST` as in the nightly builds we might later also test other things than commands also?).